applications/openshift/api-server/api_server_authorization_mode/oval/shared.xml

```diff
@@ -1,25 +1,40 @@
 <def-group>
   <definition class="compliance" id="api_server_authorization_mode" version="1">
     <metadata>
-      <title>Disable use of AlwaysAllow for the API Server Authorization Mode</title>
+      <title>Ensure authorization-mode is set to Webhook</title>
       <affected family="unix">
         <platform>multi_platform_ocp</platform>
       </affected>
-      <description>AlwaysAllow should not be configured for authentication on the master node.</description>
+      <description>authorization-mode should be set to Webhook, or not set (default is Webhook).</description>
```

Comment:
@shawndwells what about just saying `authorization-mode should be set to Webhook`? Seems wordy and confusing.

Reply:
Updated!